metainfo.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2024 Dmitrii Puzikov -->
+<component type="desktop-application">
+  <id>com.vcvrack.Rack2</id>
+  <launchable type="desktop-id">com.vcvrack.Rack2.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://vcvrack.com/eula</project_license>
+  <name>Rack 2 Free</name>
+  <summary>Virtual Eurorack Studio</summary>
+  <description>
+    <p>VCV Rack 2 is a powerful, modular synthesizer software that emulates Eurorack hardware on your computer.</p>
+    <p>It allows users to build custom synth patches using a wide range of virtual modules, mimicking the flexibility of physical modular setups.</p>
+    <p>VCV Rack 2 comes in both free and Pro versions, with the Pro version adding VST plugin integration, allowing users to use the software within their DAWs.</p>
+    <p>Its interface supports high-quality visual and sonic precision, providing musicians and sound designers with extensive creative freedom for sound exploration.</p>
+    <p>For more details, visit VCV Rack's official site.</p>
+    <p>This wrapper is not verified by, affiliated with, or supported by VCV</p>
+  </description>
+  <categories>
+    <category>Audio</category>
+    <category>Music</category>
+  </categories>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://vcvrack.com/Rack2/plugin.webp</image>
```

Comment:
webp is NOT a supported format.

Reply:
It works now, we added webp-pixbuf-loader to org.flatpak.Builder.

Reply:
nevermind then.

Reply:
Heck, I need to revert changes then :)